build

```diff
@@ -4,6 +4,11 @@
 ORG_PATH="github.com/coreos"
 REPO_PATH="${ORG_PATH}/etcd"
 export GO15VENDOREXPERIMENT="1"
+
+# Set GO_LDFLAGS="" for building with all symbols for debugging.
+if [ -z "${GO_LDFLAGS+x}" ]; then GO_LDFLAGS="-s"; else GO_LDFLAGS=""; fi
```

Comment:
`if [ -z "${GO_LDFLAGS+x}" ]; then GO_LDFLAGS="-s"; fi`

The script shouldn't override GO_LDFLAGS if GO_LDFLAGS is provided.